vernac/library.ml

```diff
@@ -298,9 +298,16 @@ let check_library_expected_name ~provenance dir library_name =
        DirPath.print library_name ++ spc () ++ str "and not library" ++
        spc() ++ DirPath.print dir ++ str ".")
 
-let error_in_intern provenance dir err = CErrors.user_err
-    Pp.(str "Error when parsing .vo (from " ++ str (fst provenance) ++ str " " ++
-        str (snd provenance) ++ str ") for library " ++ Names.DirPath.print dir ++ str ": " ++ err)
+exception InternError of { iexn : Exninfo.iexn; provenance : Intern.Provenance.t; dir : DirPath.t }
```

Comment:
```suggestion
exception InternError of { exn : exn; provenance : Intern.Provenance.t; dir : DirPath.t }
```
and raise with the info of the inner exception

Reply:
Actually I was wondering if your first request for change was motivated by wanting different backtrace info.

Should be fixed in the next revision of the PR.

I got into the habit of building this kind of reification points (mainly in Flèche), but indeed here, even if the extra backtrace is mostly noise, doesn't make so much sense.